tests/php/Forms/NumericFieldTest.php

```diff
@@ -127,6 +127,19 @@ public function testNumberTypeOnInputHtml()
         $this->assertContains('type="text"', $html, 'number type not set');
     }
 
+    public function testNullSet() {
+        $field = new NumericField('Number');
+        $field->setValue('');
+        $this->assertEquals(null, $field->Value());
```

Comment:
`assertNull` would be a more appropriate assertion here - but I won't block merge over it